src/rebar_git_resource.erl

```diff
@@ -201,7 +201,7 @@ parse_tags(Dir) ->
         {error, _} ->
             {undefined, "0.0.0"};
         {ok, Line} ->
-            case re:run(Line, "(\\(|\\s)(HEAD,\\s)tag:\\s(v?([^,\\)]+))", [{capture, [3, 4], list}]) of
+            case re:run(Line, "(\\(|\\s)(HEAD[^,],\\s)tag:\\s(v?([^,\\)]+))", [{capture, [3, 4], list}]) of
```

Comment:
Shouldn't this be `[^,]*` ? I'm not quite sure I understand how the current thing works.